rbd/system/cli/rbd_cli_misc.py

```diff
@@ -63,8 +66,8 @@ def exec_cmd(args):
     # Add Lock
     exec_cmd('rbd lock add {}/img{} 007'.format(parameters.rep_pool['val']['pool0'], iterator))
 
-    [exec_cmd('rbd lock add --shared tag {}/img{} {}'
-              .format(parameters.rep_pool['val']['pool0'], iterator - 1, lock_id))
+    [exec_cmd('rbd lock add --shared lock-tag {}/img{} {}'
```

Comment:
@Gopi-Patta lock-tag vs tag how cli changed in rhcs3 vs 4? will it run both on nautilus and luminous?

Reply:
@XDsubhash lock-tag/tag are just names which we give. I changed as per document for our better understanding and it works on both nautilus/luminous.